AccountService/src/UserInputService.java

```diff
@@ -1,24 +1,81 @@
+import java.util.ArrayList;
 import java.util.Scanner;
 
 public class UserInputService {
-    Scanner loginInput = new Scanner(System.in);
-    LoginService loginService;
-
-    public UserInputService(LoginService loginService) {
-        this.loginService = loginService;
-    }
-    public void loginInput() {
-        System.out.println("Enter username");
-        String userName = loginInput.nextLine();
-        System.out.println("Enter password");
-        String password = loginInput.nextLine();
-
-        if (loginService.verifyUser(userName, password)) {
-            System.out.println("Username and password is valid");
-        }
-        else {
-            System.out.println("Username and password is invalid");
-
-        }
-    }
+	Scanner userInput = new Scanner(System.in);
+	LoginService loginService;
+	CustomerService customerService;
+
+	public UserInputService(LoginService loginService, CustomerService customerService) {
+		this.loginService = loginService;
+		this.customerService = customerService;
+	}
+
+	public void loginInput() {
+		System.out.println("Enter username");
+		String userName = userInput.nextLine();
+		System.out.println("Enter password");
+		String password = userInput.nextLine();
+
+		if (loginService.verifyUser(userName, password)) {
+			System.out.println("Username and password is valid");
+		} else {
+			System.out.println("Username and password is invalid");
+
+		}
+	}
+
+	public void createAccountInput() {
+		ArrayList<User> newUserDatabase = customerService.userList;
+		customerService.createData();
+
+		boolean isNewUsernamePasswordValid = false;
+		String newUsername;
+		String newPassword;
+		do {
+			System.out.println("Enter username: ");
+			newUsername = userInput.nextLine();
+			System.out.println("Enter password: ");
+			newPassword = userInput.nextLine();
+			System.out.println("Enter password again: ");
+			String newPasswordCheck = userInput.nextLine();
+
+			boolean isUsernameValid = true;
+			for (User user : newUserDatabase) {
+				if (newUsername.equals(user.getUserName())) {
+					System.out.println("Username is used. Please try another one.");
+					isUsernameValid = false;
+				}
+			}
+
+			boolean isPasswordValid = true;
+			if (!newPasswordCheck.equals(newPassword)) {
+				System.out.println("Password does not match");
+				isPasswordValid = false;
+			}
```

Comment:
We can create a new method for this block of code - the method will receive `newPasswordCheck` and `newPassword` as inputs, and return a boolean as output